### ISSUE TITLE
【time】how long does it take to train？

### ISSUE THREAD

**Issue by Lz-2019317** [FudanVI/FudanOCR#10]
【time】how long does it take to train？
*********************************************
【moran、aster】how to set recognition head to moran or aster？
and Do I need to retrain every time the recognition head is set?

thank you

**Comment by JingyeChen** [comment]
【time】The model can converge better with 500 epochs. Since I am using one GTX 1080Ti with 11GB and set the batch size to 16, it takes me about two days to train the model

【moran、aster】 you could refer it to the repo of TextZoom. It will be easy for you if you are familiar with OCR :D. 
No, you can choose the best model validated by CRNN to test on other recognizers, such as moran, aster, etc.

**Comment by Lz-2019317** [comment]
【the Content-Aware Module will guide the super-resolutionmodel to generate a more distinguishable text image by backpropagation】
if i choose the different recognition head，eg：crnn、moran、aster，i will get different loss（content).And finally this lead to overall  model generate different SR text image.right? 
Therefor，i need to retrian the model by differernt recognition head.

**Comment by Lz-2019317** [comment]
when you conduct experiment of comparison of the recognition accuracy,how many times did you  train?
thank you .

**Comment by JingyeChen** [comment]
Train for many times :D

**Comment by shubhambagwari** [comment]
It will take days,
There is the file in the following path "dir\FudanOCR-main\scene-text-telescope\config\super_resolution.yaml" where you can see the number of epochs = 5000; initially, you can do 5 and predict the time.
